src/sgame/sg_bot_util.cpp

```diff
@@ -1376,6 +1377,13 @@ bool BotTargetInAttackRange( const gentity_t *self, botTarget_t target, weapon_t
 	float dist = glm::distance( muzzle, VEC2GLM( trace.endpos ) );
 	range = std::max( range, secondaryRange );
 
+	float blurrDist = ( SkillModifier( self->botMind->skillLevel ) * g_bot_distanceAccuracy.Get() ) / 2;
+	if ( random() <= 0.5 )
+	{
+		blurrDist = -blurrDist;
+	}
+	range = blurrDist;
+
```

Comment:
A lot of stuff doesn't make sense here.
- The range can be negative (!)
- The cvar is supposed to be a percentage but doesn't work like that 
- Re-rolling a random number every frame would result in starting and stopping shooting more frequently, not less.

How about saving this for another PR so we can merge the months-old stuff already?